src/lib/SCREEN/TFT/tftdisplay.cpp

```diff
@@ -119,6 +120,16 @@ constexpr uint16_t elrs_banner_bgColor[] = {
 #define SUB_PAGE_BINDING_WORD_START_X   0
 #define SUB_PAGE_BINDING_WORD_START_Y   (SCREEN_Y -  SCREEN_LARGE_FONT_SIZE)/2
 
+//LINKSTATS PAGE Definition
+#define LINKSTATS_COL_FIRST   0
+#define LINKSTATS_COL_SECOND  30
+#define LINKSTATS_COL_THIRD   100
+#define LINKSTATS_ROW_FIRST   10
+#define LINKSTATS_ROW_SECOND  25
+#define LINKSTATS_ROW_THIRD   40
+#define LINKSTATS_ROW_FOURTH  55
+#define LINKSTATS_ROW_FIFTH   70
+
```

Comment:
These can just be constants in the `displayLinkstats()` function as that is the only place they are used and it keeps the code self-contained.